templates/patterns/social_media_links/social_media_links.ui_patterns.yml

```diff
@@ -0,0 +1,32 @@
+social_media_links:
+  label: "Social media links"
+  description: "Social network icon with link of label."
+  variants:
+    horizontal:
+      label: "Horizontal"
+      description: "It will align the field items horizontally."
+    vertical:
+      label: "Vertical"
+      description: "It will align the field items vertically."
+  fields:
+    title:
+      type: "text"
+      label: "Title"
+      description: "Social media block title."
+      preview: "View European Commission on:"
+    links:
+      type: "array"
+      label: "Social links"
+      description: "List of social links."
+      preview:
+        - service: "facebook"
+          label: "Facebook"
+          url: "#"
+        - service: "twitter"
+          label: "Twitter"
+          url: "#"
+        - service: "instagram"
+          label: "Instagram"
+          url: "#"
+        - label: "See more"
```

Comment:
I don't think this link is needed, it implies some functionality that is no there on the pattern.

Reply:
It exists here https://ec.europa.eu/component-library/playground/ec/?path=/story/components-socialmediafollow--horizontal, so I think we should support it.